<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1177</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="minimumSize">
              <size>
               <width>45</width>
               <height>21</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>45</width>
               <height>21</height>
              </size>
             </property>
             <property name="text">
              <string>串口号</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="COM">
             <property name="minimumSize">
              <size>
               <width>87</width>
               <height>21</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>87</width>
               <height>21</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="minimumSize">
              <size>
               <width>45</width>
               <height>21</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>45</width>
               <height>21</height>
              </size>
             </property>
             <property name="text">
              <string>波特率</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="BautRate">
             <property name="minimumSize">
              <size>
               <width>87</width>
               <height>21</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>87</width>
               <height>21</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string>9600</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>19200</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>38400</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>57600</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>115200</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>230400</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>460800</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="openSerial_btn">
         <property name="minimumSize">
          <size>
           <width>171</width>
           <height>51</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>171</width>
           <height>51</height>
          </size>
         </property>
         <property name="text">
          <string>打开串口</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>血糖浓度：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>321</width>
         <height>511</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>321</width>
         <height>511</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="plot_btn">
       <property name="text">
        <string>实时曲线</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCustomPlot" name="plot" native="true">
       <property name="minimumSize">
        <size>
         <width>821</width>
         <height>511</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>821</width>
         <height>511</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
